src/components/Toast/README.md

```diff
@@ -0,0 +1,21 @@
+# `<Toast />`
+
+All Toast related dependencies are contained within this folder, with the exception of 3rd party libraries and a shared hook `useKeyPress`.
+
+## Directory Structure
```

Comment:
👍 I like the thoroughness here. It reminds me—I didn't include sub-components in mine. 😬